src/drivers/px4fmu/fmu.cpp

```diff
@@ -789,14 +789,10 @@ PX4FMU::cycle()
 		orb_copy(ORB_ID(actuator_armed), _armed_sub, &_armed);
 
 		/* update the armed status and check that we're not locked down */
-		bool set_armed = (_armed.armed || _armed.prearmed) && !_armed.lockdown;
```

Comment:
The pre-armed state is needed for applications which allow to move e.g. the ailerons once the button is pressed. So most likely the issue here is _armed.prearmed set in the wrong state? Is it maybe always true?